_episodes/12-cluster.md

```diff
@@ -149,6 +151,10 @@ the computer's memory. Disk is a computer's long-term storage for information it
 > {: .bash}
 {: .challenge}
 
+> ## Differences Between Nodes
+> The cluster may not be homogeneous. Many HPC clusters have a variety of nodes optimized for particular workloads. Some nodes may have larger amount of memory, or specialized resources such as Graphical Processing Units.
```

Comment:
I suggest to expand on homogeneous here. For example adding "i.e. all compute nodes consist of the same model by the manufacturer". We could leave it at this or mention RAM, CPU count, etc. ... which in turn would force us to define all of those first. Here is a great chance to add conceptual pictures, schemas or art here!

Reply:
Possibly the best solution is to just remove the first sentence, so that it just reads "Many HPC clusters have a variety ..."
If that's too terse, we could invite the learners to compare the specs on "their" node to others.

Reply:
good idea - go for it. what I typically do in my workshops is having the participants compare the specs of a node with those of their laptop. this may take time as finding the processor type etc in different OSes can be challenging. But I feel that its a worthwhile task as they can relate better to what they are working with. In the future, we might even have them compare with the specs of their mobile phone or so.